app/components/base/Card.tsx

```diff
@@ -178,12 +186,19 @@ type Props = {
   isFlex?: boolean;
   icon?: string;
   badge?: string;
-  productName?: string;
+  product?: "BALANCE" | "CONNECT";
```

Comment:
```suggestion
  product?: keyof typeof productDisplayText;
```
Not required, but will make it so TS tracks `productDisplayText`, rather than explicitly putting the values here

Reply:
Nice! I like this better. Thanks! :)